pallets/restricted-tokens/Cargo.toml

```diff
@@ -12,16 +12,16 @@ version = "0.1.0"
 targets = ["x86_64-unknown-linux-gnu"]
 
```

Comment:
According to my search, this pallet has quite a number of dependencies which don't make use of workspace inheritance.